dialogues/riches/dining_room.json

```diff
@@ -16,5 +16,13 @@
   "wall_cases": {
     "content": "The display cases along the wall feature mostly old newspapers reporting the miraculous survival of a man who got struck by ball lightning. There are some panels speculating on the nature of the phenomenon, hinting at its spiritual or divine origin.",
     "next": "first"
+  },
+  "maybe_rift": {
```

Comment:
Needs double fixing. 

- First of all it should be the other way around. The rift should be in the study, leading to the dining room. 
- The syntax is off. It needs to be ` "next": ["rags/study", "first"]`